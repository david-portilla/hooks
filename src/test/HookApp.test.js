import React from 'react'
import { shallow, configure } from 'enzyme';
import { HookApp } from '../HookApp'
import Adapter from 'enzyme-adapter-react-16';

describe('testing <HookApp />', () => {

  configure({ adapter: new Adapter() });
  test('should render properly', () => {
    const wrapper = shallow(<HookApp />)
    expect(wrapper).toMatchSnapshot()
  })

})
