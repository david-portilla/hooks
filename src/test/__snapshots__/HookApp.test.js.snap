// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing <HookApp /> should render properly 1`] = `ShallowWrapper {}`;
